tests/httpClient.test.ts

```diff
@@ -27,7 +27,7 @@ describe('createHeadersWithResolvedToken()', () => {
       Authorization: 'Bearer abc',
     };
     await expect(httpClient.createHeadersWithResolvedToken(headers)).rejects.toThrow(
```

Comment:
Hmm maybe we could even do the typeof check here. curious if it'll work

Reply:
changed, works :)